packages/frontity/src/__tests__/create-package.cli.test.ts

```diff
@@ -1,19 +1,17 @@
-import createPackage from "../create-package";
+import createPackage from "../cli/create-package";
 import inquirer from "inquirer";
-import * as utils from "../../utils";
+import * as utils from "../utils";
 
-jest.mock("../../functions/create-package");
 jest.mock("inquirer");
-jest.mock("events");
-jest.mock("../../utils");
 
 const mockedInquirer = inquirer as jest.Mocked<typeof inquirer>;
-const mockedUtils = utils as jest.Mocked<typeof utils>;
+const mockedUtils = utils as any;
```

Comment:
It's just easier this way to fix: `Cannot assign to 'isFrontityProjectRoot' because it is a read-only property`